src/Import/include/FMI3/fmi3_import.h

```diff
@@ -244,17 +244,6 @@ FMILIB_EXPORT fmi3_import_type_definitions_t* fmi3_import_get_type_definitions(f
 /** \brief Get a list of all the unit definitions in the model. */
 FMILIB_EXPORT fmi3_import_unit_definitions_t* fmi3_import_get_unit_definitions(fmi3_import_t* fmu);
 
-/** \brief Get the variable with the same value reference that is not an alias*/
-FMILIB_EXPORT fmi3_import_variable_t* fmi3_import_get_variable_alias_base(fmi3_import_t* fmu, fmi3_import_variable_t* v);
```

Comment:
Are we sure on removing this, I would think it could be worthwhile to have a such that function that returns the base variable, given an alias. Sure, one can do it via `fmi3_import_get_variable_by_name(fmi3_import_get_alias_variable_name(alias))`, but that feels stupidly expensive for large models.

Or is this functionality available somewhere else and I missed it?

Reply:
Ok, can always add it later on if useful/needed.